### ISSUE TITLE
Add example for loaders?

### ISSUE THREAD

**Issue by kulakowka** [matthewmueller/graph.ql#8]
@matthewmueller can you add simple example for data fetching?

``` javascript
// an object of promises that fetch actual data
var loaders = require('./loaders')
...
Person: {
    films(person) {
      return loaders.film.loadMany(person.films)
    }
  },
```

How should look like loader? Should method `loadMany` return the usual promise?

**Comment by tgriesser** [comment]
@kulakowka yep, just return the usual promise as you would in a normal graphql `resolve` - check out the [test suite](https://github.com/matthewmueller/graph.ql/blob/ab768e1d3abd2a26b2010e9acf32f28a2db7bd1c/test/implementation.js#L70-L121) for more examples
